src/Conf.ml

```diff
@@ -877,8 +877,12 @@ module Formatting = struct
       "Maximum offset ($(docv) columns) added to a new line in addition to \
        the offset of the previous line."
     in
+    (* creating a fresh formatter in case the value of max-indent has been
+       changed for stdout. *)
+    let fs = Format.formatter_of_buffer (Buffer.create 4) in
+    let none = Int.to_string (Format.pp_get_max_indent fs ()) in
```

Comment:
What do you think about pulling this into a top-level `default_max_indent_value`? also `4` looks like it's meaningful here - if it's not I suggest passing `0`.

Reply:
done.